### PR TITLE
Fix rclone move command

### PR DESCRIPTION
**Pull request by carleeno**
Ok hopefully this is a better start:

Removed for loop and running a single move command on the SavedClips directory. Rclone docs explain that a move command moves the contents of the source directory, not the source directory itself.
Also getting file count for push notification, instead of directory count. This only works with Tesla firmware since 2019.5...

### CODE REVIEW

File: run/rclone_archive/archive-clips.sh [hunk 1]
```diff
@@ -4,20 +4,15 @@ log "Moving clips to rclone archive..."
...
-for file_name in "$CAM_MOUNT"/TeslaCam/saved* "$CAM_MOUNT"/TeslaCam/SavedClips/*; do
-  [ -e "$file_name" ] || continue
-  log "Moving $file_name ..."
-  rclone --config /root/.config/rclone/rclone.conf move "$file_name" "$drive:$path" >> "$LOG_FILE" 2>&1 || echo ""
-  log "Moved $file_name."
-  NUM_FILES_MOVED=$((NUM_FILES_MOVED + 1))
-done
+FILE_COUNT=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -type f | wc -l)
+
+rclone --config /root/.config/rclone/rclone.conf move "$CAM_MOUNT"/TeslaCam/SavedClips "$drive:$path" --create-empty-src-dirs --delete-empty-src-dirs >> "$LOG_FILE" 2>&1 || echo ""
```
Comment (marcone) [inline run/rclone_archive/archive-clips.sh:9]: still wondering about that `--create-empty-src-dirs` option. Do you really want to replicate empty directories in the archive?

**Review by marcone — COMMENTED** [review]
Didn't you also want to report the number of directories?

File: run/rclone_archive/archive-clips.sh (hunk 1 above)
Comment (carleeno) [reply run/rclone_archive/archive-clips.sh:9]: @marcone I thought about it, and yes I think it's wise, because if Tesla one day has a bug where it is creating empty directories (or if the script is somehow causing directories to be empty) I'd like to at least have the empty dir move to the archive so I see it and know there is a problem. Otherwise there would be nothing to indicate there is a problem.
Also it allows testing with empty folders, that way if anybody manually creates an empty folder using their PC to test the upload works, it will behave as expected, copying anything that exists in the SavedClips directory.

**Comment by carleeno** [comment]
@marcone PR 189 has the code to notify number of directories as well as files. This is in archiveloop so it works for all 3 archive methods.

File: run/rclone_archive/archive-clips.sh (hunk 1 above)
Comment (carleeno) [reply run/rclone_archive/archive-clips.sh:9]: @marcone (sorry to keep tagging you but I'm not sure if you noticed these replies earlier)
Another note about the create empty dirs, is that since we also have --delete-empty-src-dirs it will delete an empty directory from the source once all the files are moved over, so normally there won't be empty directories accumulating (previously this was a problem)
